service_container/tags.rst

```diff
@@ -16,8 +16,7 @@ to be used for a specific purpose. Take the following example:
 
         # app/config/services.yml
         services:
-            app.twig_extension:
-                class: AppBundle\Twig\AppExtension
+            AppBundle\Twig\AppExtension:
                 public: false
```

Comment:
I hesitated to use `_defaults` here to be able to remove this line and use the short syntax, wdyt?